arm-software/embedded/llvmlibc-samples/src/baremetal-semihosting-aarch64-llvmlibc/hello.c

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2020-2025, Arm Limited and affiliates. 
+ *
+ * Part of the Arm Toolchain project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception */
+
+#include <stdio.h>
+
+int main(void) {
```

Comment:
suggestion:

The v6-m sample, while somewhat contrived, makes use of the heap, string and math library too. For example the bit in the linker script about setting up the heap location won't get used in this sample.

Given the name of the sample is the same apart from AArch64, is it worth using the same code on AArch64?